### ISSUE TITLE
Dual IP stack support

### ISSUE THREAD

**Issue by michaelklishin** [rabbitmq/rabbitmq-management#64]
See [this mailing list thread](https://groups.google.com/forum/#!topic/rabbitmq-users/9C1-Oqgr9WU). This is a good thing to take care of as part of #63.

**Comment by essen** [comment]
There's nothing to do for that one, once #63 is finished. The issue was that Mochiweb was forcing IPv6 when an IPv6 address was given. Cowboy does no such thing, and just passes on whatever TCP options you configured.

There may be a need to update configuration to keep the previous result, if it was desired, however. We will need to warn about this configuration change.

**Comment by michaelklishin** [comment]
@essen thank you for the update. I think updating the docs and using user config as is without forcing anything is a reasonable way to proceed.

**Comment by michaelklishin** [comment]
This is no longer relevant after #63 is fixe.
